logrusr.go

```diff
@@ -17,17 +17,25 @@ const (
 )
 
 type logrusr struct {
-	name   []string
-	level  int
-	logger logrus.FieldLogger
+	name             []string
+	level            int
+	logger           logrus.FieldLogger
+	defaultFormatter func(interface{}) string
 }
 
 // NewLogger will return a new logr.Logger from a logrus.FieldLogger.
 func NewLogger(l logrus.FieldLogger, name ...string) logr.Logger {
+	return NewLoggerWithFormatter(l, nil, name...)
+}
+
+// NewLoggerWithFormatter will return a new logr.Logger from a logrus.FieldLogger that uses provided function to
+// format complex data types.
```

Comment:
Just consistent line length, wrap  the comments at 80 columns like the rest.

```suggestion
// NewLoggerWithFormatter will return a new logr.Logger from a
// logrus.FieldLogger that uses provided function to format complex data types.
```